v3/docs/gs/params.md

```diff
@@ -313,8 +320,7 @@ legal values, and defaults.
       },
       "compression": {
         "value": {
-          "algorithm": "lz4 | none",
-          "min_length": integer
+          "algorithm": "lz4 | none"
         }
```

Comment:
Change to these params too:
```
	"compression":	{
		"default":	"off",
		"algorithm":	"lz4"
	}
```

Reply:
Current global config params:
```
{
        "logging":      {
                "enabled":      true,
                "destination":  "syslog",
                "path": "hse.log",
                "level":        6,
                "squelch_ns":   0
        },
        "vlb_cache_sz": 4294967296,
        "c0kvs_ccache_sz_max":  4294967296,
        "c0kvs_ccache_sz":      4294967296,
        "c0kvs_cheap_sz":       134217728,
        "workqueue_tcdelay":    1000,
        "workqueue_idle_ttl":   300,
        "perfc":        {
                "level":        2
        },
        "rest": {
                "enabled":      true,
                "socket_path":  "/run/user/1145546/hse-115409.sock"
        }
}
```

Reply:
Current KVDB config params:
```
{
        "read_only":    false,
        "perfc":        {
                "level":        2
        },
        "perfc_enable": 2,
        "c0_debug":     0,
        "c0_diag_mode": false,
        "c0_ingest_width":      37,
        "txn_timeout":  300000,
        "cndb_compact_hwm_pct": 1.6148387969641e-311,
        "csched_policy":        0,
        "csched_debug_mask":    0,
        "csched_samp_max":      150,
        "csched_lo_th_pct":     25,
        "csched_hi_th_pct":     75,
        "csched_leaf_pct":      90,
        "csched_gc_pct":        67,
        "csched_max_vgroups":   64,
        "csched_lscat_runlen_max":      3,
        "csched_qthreads":      2211925001477,
        "csched_rspill_params": 0,
        "csched_leaf_comp_params":      0,
        "csched_leaf_len_params":       0,
        "csched_node_min_ttl":  17,
        "durability":   {
                "enabled":      true,
                "interval_ms":  100,
                "replay":       {
                        "force":        false
                },
                "size_bytes":   8388608,
                "buffer":       {
                        "size": 4096,
                        "managed":      false
                },
                "throttling":   {
                        "threshold":    {
                                "low":  13,
                                "high": 87
                        }
                },
                "mclass":       "auto"
        },
        "throttle_disable":     false,
        "throttle_update_ns":   25000000,
        "throttle_debug":       0,
        "throttle_debug_intvl_s":       300,
        "throttling":   {
                "init_policy":  "medium"
        },
        "throttle_burst":       1048576,
        "throttle_rate":        10485760,
        "txn_wkth_delay":       60000,
        "c0_maint_threads":     3,
        "c0_ingest_threads":    3,
        "cn_maint_threads":     32,
        "cn_io_threads":        17,
        "keylock_tables":       761,
        "mclass_policies":      [],
        "storage":      {
                "capacity":     {
                        "file": {
                                "max_size":     2048,
                                "count":        32
                        },
                        "mblock":       {
                                "size": 32
                        },
                        "path": "/var/hse/test/hse/capacity",
                        "directio":     {
                                "enabled":      true
                        }
                },
                "staging":      {
                        "file": {
                                "max_size":     0,
                                "count":        0
                        },
                        "mblock":       {
                                "size": 0
                        },
                        "path": "",
                        "directio":     {
                                "enabled":      true
                        }
                },
                "pmem": {
                        "file": {
                                "max_size":     0,
                                "count":        0
                        },
                        "mblock":       {
                                "size": 0
                        },
                        "path": "",
                        "directio":     {
                                "enabled":      true
                        }
                }
        }
}
```

Reply:
Current kvs config params:
```
{
        "prefix":       {
                "length":       0
        },
        "kvs_ext01":    0,
        "suffix":       {
                "length":       0
        },
        "kvs_cursor_ttl":       1500,
        "transactions": {
                "enabled":      false
        },
        "perfc":        {
                "level":        2
        },
        "cn_split_size":        19,
        "cn_compact_vblk_ra":   262144,
        "cn_compact_vra":       131072,
        "cn_compact_kblk_ra":   524288,
        "cn_capped_ttl":        9000,
        "cn_capped_vra":        524288,
        "cn_cursor_vra":        0,
        "cn_cursor_kra":        false,
        "cn_cursor_seq":        0,
        "cn_mcache_wbt":        0,
        "cn_mcache_vmax":       4096,
        "cn_mcache_kra_params": 0,
        "cn_mcache_vra_params": 0,
        "cn_diag_mode": false,
        "cn_maint_disable":     false,
        "cn_bloom_create":      true,
        "cn_bloom_preload":     false,
        "cn_bloom_prob":        10000,
        "cn_bloom_capped":      0,
        "cn_compaction_debug":  0,
        "cn_maint_delay":       1000,
        "cn_close_wait":        false,
        "cn_kcachesz":  1048576,
        "capped_evict_ttl":     120,
        "read_only":    false,
        "mclass":       {
                "policy":       "capacity_only"
        },
        "compression":  {
                "default":      "off",
                "algorithm":    "lz4"
        }
}
```

Reply:
I will check these against the docs to see if there are other changes.

Reply:
@alexttx , from the above, it looks like logging.structured has been removed.  Is that the case?

Reply:
Yes, it has been removed.